src/onefetch/cli.rs

```diff
@@ -349,6 +358,16 @@ impl Cli {
             Vec::new()
         };
 
+        let bot_exclude_pattern = if matches.is_present("no-bots") {
+            if let Some(pattern) = matches.value_of("no-bots") {
+                Some(String::from(pattern))
+            } else {
+                Some(String::from(r".*\[bot\].*"))
+            }
+        } else {
+            None
+        };
```

Comment:
```suggestion
        let bot_exclude_pattern = matches.is_present("no-bots").then(
            || matches.value_of("no-bots").map_or(r"\[bot\]".to_string(), String::from)
        )
```
I think this would work. Been a while since I've used Rust, so the borrow checker might not like the closer, though :shrug:

As a side note, for regex, I think `.*\[bot\].*` and `\[bot\]` would match the same things, since `.` matches everything but a newline AFAIK. Just that `.*\[bot\].*` would match the whole string, and `\[bot\]` would only match `[bot]` within the string.

This is totally *not* a necessary change, just comes down to preferred coding style :smiley:

Reply:
You're absolutely right, I thought the Regex had to match the entire string to return true :man_facepalming: